### PR TITLE
Update LAB[PL-900]_M02Lab01_Data_Model.md

### PR DESCRIPTION
**Pull request by excelwithallison**
Updated terminology to use 'table' and 'column' in place of 'entity' and 'field'

# Module: 00
## Lab/Demo: 00

Fixes # .

Changes proposed in this pull request:

-
-
-

### CODE REVIEW

File: Instructions/Labs/LAB[PL-900]_M02Lab01_Data_Model.md [hunk 1]
```diff
@@ -308,6 +305,7 @@ In this task you will import a solution that contains the Power Automate flow re
 3. If **Status** is **Off**, select **...** next to **Import Data** then select **Turn On**.
 
    > **Important:** If you receive an error message, verify that the entities and fields you created match the instructions above.
```
Comment (excelwithallison) [inline Instructions/Labs/LAB[PL-900]_M02Lab01_Data_Model.md:307]: I had to update the flow to remove the bc_building@odata reference.

File: Instructions/Labs/LAB[PL-900]_M02Lab01_Data_Model.md [hunk 2]
```diff
@@ -308,6 +305,7 @@ In this task you will import a solution that contains the Power Automate flow re
 3. If **Status** is **Off**, select **...** next to **Import Data** then select **Turn On**.
 
    > **Important:** If you receive an error message, verify that the entities and fields you created match the instructions above.
+
```
Comment (excelwithallison) [inline Instructions/Labs/LAB[PL-900]_M02Lab01_Data_Model.md:308]: There are a few more refences to entity and field below that need changed too.

**Review by excelwithallison — COMMENTED** [review]
I have fixed the flow, so was able to finish the lab, but don't know how to edit the pull request to update the remaining steps.

File: Instructions/Labs/LAB[PL-900]_M02Lab01_Data_Model.md (hunk 1 above)
Comment (chandlersyal) [reply Instructions/Labs/LAB[PL-900]_M02Lab01_Data_Model.md:307]: Hi Allison, can you provide more info here? Why did you need to remove the reference? Was it not working properly?

Comment (excelwithallison) [reply Instructions/Labs/LAB[PL-900]_M02Lab01_Data_Model.md:307]: I did end up getting it to work correctly, so not sure what the problem was the first time, so we can ignore this comment. 

I did just have a student have an issue where the error said: 
"There are missing dependencies. Install the following solutions before installing this one: "Active"

image

So any help on that error would be appreciated.

Comment (chandlersyal) [reply Instructions/Labs/LAB[PL-900]_M02Lab01_Data_Model.md:307]: I would verify that they have all of the entities/tables and fields/columns named exactly right.

**Comment by chandlersyal** [comment]
Closing, as these terminology updates were made with the March 2021 update. Thanks!

**Pull request closed without merging**